src/collections/views/CollectionCreate.tsx

```diff
@@ -1,25 +1,27 @@
 import React from "react";
+import { useIntl } from "react-intl";
 
 import { WindowTitle } from "@saleor/components/WindowTitle";
 import useNavigator from "@saleor/hooks/useNavigator";
 import useNotifier from "@saleor/hooks/useNotifier";
-import i18n from "../../i18n";
 import { getMutationState, maybe } from "../../misc";
 import { CollectionCreateInput } from "../../types/globalTypes";
 import CollectionCreatePage from "../components/CollectionCreatePage/CollectionCreatePage";
 import { TypedCollectionCreateMutation } from "../mutations";
 import { CreateCollection } from "../types/CreateCollection";
 import { collectionListUrl, collectionUrl } from "../urls";
 
-export const CollectionCreate: React.StatelessComponent<{}> = () => {
+export const CollectionCreate: React.FC = () => {
   const navigate = useNavigator();
   const notify = useNotifier();
+  const intl = useIntl();
 
   const handleCollectionCreateSuccess = (data: CreateCollection) => {
     if (data.collectionCreate.errors.length === 0) {
       notify({
-        text: i18n.t("Created collection", {
-          context: "notification"
+        text: intl.formatMessage({
+          defaultMessage: "Created collection",
```

Comment:
We should fix this message. Either to say "Collection created" or something longer like "Collection was created successfully". Personally, I'd drop all notifications where the result is obvious (either you get redirected or the screen changes significantly) unless we offer an undo option ("Product Foo was deleted. [UNDO]?").

Reply:
After consultation with @pwgryglak, we decided that notifications stay and texts will be improved.